modules/services/taginfo.js

```diff
@@ -222,7 +222,7 @@ export default {
         this.keys(params, function(err, data) {
             if (err) return;
             data.forEach(function(d) {
-                if (d.value === 'opening_hours') return;  // exception
+                if (d.value === 'opening_hours') return;  //exception
                 _popularKeys[d.value] = true;
```

Comment:
I was wondering how this service excludes `name`, since it isn’t explicitly listed in `_popularKeys`. It turns out that `name` happens to be one of the 100 most popular keys that we add to the `_popularKeys` here. (There’s some additional explanation at https://github.com/openstreetmap/iD/issues/7485#issuecomment-605681756 that I had forgotten.)